### PR TITLE
perf: reorganize a few imports to improve bundle size

### PR DESCRIPTION
**Pull request by kevcodez**
- Avoid loading a huge constants file when just importing smol constants from SQLEditor*
- Avoid unnecessarily loading a bunch of dependencies through helpers file

Reduces main bundle size by ~5%

### CODE REVIEW

File: apps/studio/lib/password-strength.ts [hunk 1]
```diff
@@ -0,0 +1,46 @@
+import { post } from 'lib/common/fetch'
+import { API_URL, DEFAULT_MINIMUM_PASSWORD_STRENGTH, PASSWORD_STRENGTH } from 'lib/constants'
+import { toast } from 'react-hot-toast'
+
+export default async function passwordStrength(value: string) {
```
Comment (kevcodez) [inline apps/studio/lib/password-strength.ts:5]: Just moved from helpers, no code changes

File: apps/studio/lib/uuid.ts [hunk 2]
```diff
@@ -0,0 +1,7 @@
+import { v4 as _uuidV4 } from 'uuid'
+
+const uuidv4 = () => {
```
Comment (kevcodez) [inline apps/studio/lib/uuid.ts:3]: Just moved from helpers, no code changes

File: apps/studio/data/database/database-table-definition.ts [hunk 3]
```diff
@@ -0,0 +1,569 @@
+import minify from 'pg-minify'
+
+// Derived from https://github.com/MichaelDBA/pg_get_tabledef
+// NOTE: when updating, \n must be replaced with \\n in the SQL below
+
+export const CREATE_PG_GET_TABLEDEF_SQL = minify(
```
Comment (kevcodez) [inline apps/studio/data/database/database-table-definition.ts:6]: Just moved here, no code changes

File: apps/studio/components/interfaces/SQLEditor/SQLEditor.queries.ts [hunk 4]
```diff
@@ -0,0 +1,1518 @@
+import type { SQLTemplate } from './SQLEditor.types'
+
+export const SQL_TEMPLATES: SQLTemplate[] = [
```
[the remark this answers is not in the document]
Comment (alaister) [reply apps/studio/components/interfaces/SQLEditor/SQLEditor.queries.ts:3]: good to know – did not want to check this 😅

**Review by alaister — APPROVED** [review]
LGTM!

**Pull request merged**